.pre-commit-config.yaml

```
@@ -0,0 +1,28 @@
+        args: [--markdown-linebreak-ext=md]
+        exclude: "^thirdparty"
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+        exclude: "^thirdparty"
+      - id: check-yaml
+
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.9.0
+    hooks:
+      - id: ruff
+        args: [ --fix ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)